src/PKSim.Presentation/Presenters/Parameters/TableParameterPresenter.cs

```diff
@@ -143,4 +148,28 @@ private ValuePoint valuePointFrom(ValuePointDTO valuePointDTO)
 
       public override void RemovePoint(ValuePointDTO pointToRemove) => _tableFormulaDTO.AllPoints.Remove(pointToRemove);
    }
+
+   public class TableParameterPresenter : TableParameterPresenter<ITableFormulaView>, ITableParameterPresenter
+   {
+      public TableParameterPresenter(ITableFormulaView view, IParameterTask parameterTask, IFormulaFactory formulaFactory, ICloner cloner) :
+         base(view, parameterTask, formulaFactory, cloner)
+      {
+         // default import function disabled when context is not specified
+         view.ImportVisible = CanImport;
+      }
+
+      protected override DataRepository ImportTablePoints()
+      {
+         // default import function disabled when context is not specified
+         throw new System.NotImplementedException();
+      }
+
+      protected override TableFormula TablePointsToTableFormula(DataRepository importedTablePoints)
+      {
+         // default import function disabled when context is not specified
+         throw new System.NotImplementedException();
+      }
+
+      public override bool CanImport => false;
+   }
```

Comment:
The class went missing as I moved presenters around to get table formula view in MoBi. 

This particular presenter did not allow import from Excel